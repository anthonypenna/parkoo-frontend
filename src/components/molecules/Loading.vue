<template>
  <Overlay>
    <Loader />
  </Overlay>
</template>

<script lang="ts">
import Vue from "vue";
import Overlay from "@/components/atoms/Overlay.vue";
import Loader from "@/components/atoms/Loader.vue";

export default Vue.extend({
  name: "Loading",
  components: {
    Overlay,
    Loader,
  },
});
</script>
