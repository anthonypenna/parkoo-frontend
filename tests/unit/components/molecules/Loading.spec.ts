import Loading from '@/components/molecules/Loading.vue'
import { shallowMount } from '@vue/test-utils'

describe('<Loading />', () => {
  it('should render correctly', () => {
    const wrapper = shallowMount(Loading)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
