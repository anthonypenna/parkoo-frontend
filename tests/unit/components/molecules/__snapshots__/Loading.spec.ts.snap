// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should render correctly 1`] = `
<overlay-stub>
  <loader-stub></loader-stub>
</overlay-stub>
`;
